docs/requirements.txt

```diff
@@ -24,72 +24,68 @@ idna==2.9
     # via requests
 imagesize==1.4.1
     # via sphinx
-jinja2==2.11.1
-    # via
-    #   myst-parser
-    #   sphinx
-lxml==4.5.2
+lxml==5.2.1
     # via sphinx-material
-markdown-it-py==2.1.0
+markdown-it-py==3.0.0
     # via
     #   mdit-py-plugins
     #   myst-parser
 markupsafe==2.0.1
     # via
     #   -r requirements.in
     #   jinja2
-mdit-py-plugins==0.3.5
+mdit-py-plugins==0.4.0
     # via myst-parser
 mdurl==0.1.1
     # via markdown-it-py
-myst-parser==1.0.0
+myst-parser==2.0.0
     # via -r requirements.in
-packaging==20.3
+packaging==21.0
     # via sphinx
 pillow==9.2.0
     # via -r requirements.in
-pygments==2.5.2
+pygments==2.17.2
     # via sphinx
 pyparsing==2.4.6
     # via packaging
 python-slugify[unidecode]==4.0.1
     # via sphinx-material
 pytz==2019.3
     # via babel
-pyyaml==6.0
+pyyaml==6.0.1
     # via myst-parser
-requests==2.23.0
+requests==2.25.0
     # via sphinx
 six==1.14.0
     # via packaging
 snowballstemmer==2.2.0
     # via sphinx
 soupsieve==2.0.1
     # via beautifulsoup4
-sphinx==5.0.2
+sphinx==7.3.7
```

Comment:
Let's add sphinx to `requirements.in` with the expected version, since this is the primary dependency